src/App.jsx

```diff
@@ -3,26 +3,49 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then((todos) => {
+        this.setState({ todos: todos.data });
+      });
+  }
+
   render() {
     const { todos, selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            selectedUserId={selectedUserId}
+            selectedUser={(userId) => {
+              if (this.state.selectedUserId !== userId) {
+                this.setState({ selectedUserId: userId });
+              }
+            }}
+          />
         </div>
 
         <div className="App__content">
           <div className="App__content-container">
             {selectedUserId ? (
-              <CurrentUser userId={selectedUserId} />
+              <CurrentUser
+                userId={selectedUserId}
+                selectedUser={(userId) => {
```

Comment:
selectedUser is a function, the function should be named as some action
```suggestion
                selectUser={(userId) => {
```